webapp/channels/src/components/admin_console/server_logs/logs.tsx

```diff
@@ -144,16 +161,28 @@ export default class Logs extends React.PureComponent<Props, State> {
                         <FormattedMessage {...messages.bannerDesc}/>
                     </div>
                 </div>
-                <button
-                    type='submit'
-                    className='btn btn-primary'
-                    onClick={this.reloadPlain}
-                >
-                    <FormattedMessage
-                        id='admin.logs.ReloadLogs'
-                        defaultMessage='Reload Logs'
-                    />
-                </button>
+                <div className='banner-buttons'>
+                    <button
+                        type='submit'
+                        className='btn btn-primary'
+                        onClick={this.reloadPlain}
+                    >
+                        <FormattedMessage
+                            id='admin.logs.ReloadLogs'
+                            defaultMessage='Reload Logs'
+                        />
+                    </button>
+                    <button
+                        type='submit'
+                        className='btn btn-primary'
+                        onClick={this.downloadLogs}
+                    >
+                        <FormattedMessage
+                            id='admin.logs.DownloadLogs'
+                            defaultMessage='Download Logs'
+                        />
+                    </button>
```

Comment:
One thing to note here is that in plain view, the download button "only" downloads the current page of logs. On the formated JSON page, the download button seams to download everything.

Can we bring both modes into feature parity or at least make it clear in the UI which logs are getting downloaded?

Reply:
So should I make it download current page logs for both or download all logs for both plain and JSON formatted.

Reply:
@hanzei I agree, I think all logs should be downloaded.

Thanks for the PR @Aryakoste !

Reply:
Done.